lib/routes/users.js

```diff
@@ -49,7 +51,9 @@ function routes(router) {
   router.all('/:name', responses.denyTrailingSlashOnId);
   router.get('/:name', responses.getRouteFromDB);
   router.put('/:name', responses.denyReferenceAtRoot);
+  router.put('/:name', withAuthLevel(authLevels.ADMIN));
```

Comment:
I think that when we POST the same user info multiple times, *BOTH* users end up existing. This hasn't been a problem in practice, but will become one with the new auth system.
Can you add checking-for-dupes to the user creation route handler?